### PR TITLE
Implement Tencent Video Player Backend

### PR DESCRIPTION
**Pull request by sendr**
- MWP version of the [Tencent video player](https://intl.cloud.tencent.com/document/product/266/38098), which provides a simple play of the tencent videos.
- Fixed tests, connected with python3 changes
- Refactoring of `BasePlayer` to have the ability to add own backend's validation messages.

The feature was implemented under the ticket
[HARROW-976](https://youtrack.raccoongang.com/issue/HARROW-976) [Tencent Video Player] Implement Tencent super player video-xblock (#477)

### CODE REVIEW

File: video_xblock/backends/tencent.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    def validate_data(self, validation, data):
+        """
+        Validate app id value which is mandatory.
+
+        Attributes:
+            validation (xblock.validation.Validation): Object containing validation information for an xblock instance.
+            data (xblock.internal.VideoXBlockWithMixins): Object containing data on xblock.
+        """
+        # Validate provided app id
```
[the remark this answers is not in the document]
Comment (dyudyunov) [reply video_xblock/backends/tencent.py:47]: and already exists in the method docstring

File: video_xblock/video_xblock.py [hunk 2]
```diff
@@ -103,6 +105,13 @@ class VideoXBlock(
         scope=Scope.content,
     )
 
+    app_id = String(
+        default='',
+        display_name=_('AppID'),
+        help=_('Your AppID of the VOD account. E.g. 1400329073'),
```
Comment (dyudyunov) [inline video_xblock/video_xblock.py:111]: What is the `VOD account`?))

Comment (dyudyunov) [reply video_xblock/video_xblock.py:111]: aha! it stands for `video on demand` )

**Review by dyudyunov — APPROVED** [review]
Great work! 

I've left a few minor comments inline, but overall LGTM)
@idegtiarov please take a look too

**Review by dyudyunov — APPROVED** [review]
👍

**Review by cmltaWt0 — APPROVED** [review]

**Pull request merged**